pentagon/migration/migrations/migration_2_5_0.py

```diff
@@ -0,0 +1,44 @@
+from pentagon import migration
+from pentagon.migration import *
+from pentagon.component import core, inventory
+from pentagon.helpers import merge_dict
+
+import re
+
+
+class Migration(migration.Migration):
+    _starting_version = '2.5.0'
+    _ending_version = '2.6.0'
+
+    def run(self):
+        for item in self.inventory:
+            inventory_path = "inventory/{}".format(item)
+            logging.debug(
+                'Processing Inventory Item: {}'
+                .format(inventory_path)
+            )
+
+            # Update version of VPC TF module
+            aws_vpc_file = "{}/terraform/aws_vpc.tf".format(inventory_path)
+            if os.path.exists(aws_vpc_file):
+                aws_vpc_file_content = self.get_file_content(aws_vpc_file)
+                aws_vpc_file_content = re.sub(r'terraform-vpc.git\?ref=v\d+\.\d+.\d+', 'terraform-vpc.git?ref=v3.0.0', aws_vpc_file_content)
+                aws_vpc_file_content = re.sub(r'\n\s*aws_secret_key\s+=.+', '', aws_vpc_file_content)
+                aws_vpc_file_content = re.sub(r'\n\s*aws_access_key\s+=.+', '', aws_vpc_file_content)
+                self.overwrite_file(aws_vpc_file, aws_vpc_file_content)
+                logging.info('Terraform VPC module updated to 3.0.0 in {}'.format(item))
+
+            # Remove TF AWS provider variables from secrets. No longer referenced directly in VPC module.
+            secret_file = "{}/config/private/secrets.yml".format(inventory_path)
+            if os.path.exists(secret_file):
+                secrets_file_content = self.get_file_content(secret_file)
```

Comment:
No change required but in the `Migration.YamlEditor` may have been a little easier than `re.sub` https://github.com/reactiveops/pentagon/blob/master/pentagon/migration/__init__.py#L106

If you evaluated that and chose not to, then ignore me.

Reply:
@ejether I did start with `YamlEditor`. I ran into two issues. The first was that it seemed to strip comments. And second for at least one of the secrets files I tested it on it seemed to reorder keys. Secrets files seem to be a little more varied than vars.yaml files. I was hesitant to risk breaking secrets file formatting. I want to revisit it though to understand it better going forward. Thanks for reviewing and merging.